rules/php73/tests/Rector/FuncCall/RemoveMissingCompactVariableRector/Fixture/compact_over_array.php.inc

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Rector\Php73\Tests\Rector\FuncCall\RemoveMissingCompactVariableRector\Fixture;
+
+function run()
+{
+    $foo = 'foo';
+    $bar = 'bar';
+
+    compact(['foo', 'bar'], ['baz']);
+    compact(['foo'], ['bar', 'baz']);
+    compact(['foo'], ['bar'], 'baz');
+}
```

Comment:
Function run() does not describe the case very well.
If possible, use fixture named classes here, e.g. `CompactOverArray`. See other fixture files.